packages/ui/src/components/_utils/analytics.ts

```diff
@@ -96,6 +103,11 @@ export function sendAnalyticsClickEvent<Component extends ComponentNames['click'
     }
 }
 
+/**
+ * Send analytics click event
```

Comment:
It’s not even click. Remove incorrect docs. They don’t make any sense and do not add any knowledge

Reply:
I'll remove comments to let you put proper description by yourself, since I'm not deep in purposes of bridge & UI packages